src/Compilers/VisualBasic/Test/Symbol/SymbolsTests/StaticAbstractMembersInInterfacesTests.vb

```diff
@@ -1611,13 +1715,46 @@ BC30275: 'Shared' is not valid on an interface event declaration.
         End Sub
 
         <Fact>
-        Public Sub ImplementAbstractStaticEvent_01()
+        Public Sub DefineVirtualStaticEvent_01()
+
+            Dim source1 =
+<compilation>
+    <file name="c.vb"><![CDATA[
+Interface I1
+    Overridable Shared Event E1 As System.Action
+End Interface
+]]></file>
+</compilation>
+
+            Dim comp1 = CreateCompilation(source1, targetFramework:=_supportingFramework)
+            comp1.AssertTheseDiagnostics(
+<errors>
+BC30243: 'Overridable' is not valid on an event declaration.
+    Overridable Shared Event E1 As System.Action
+    ~~~~~~~~~~~
+BC30275: 'Overridable' is not valid on an interface event declaration.
+    Overridable Shared Event E1 As System.Action
+    ~~~~~~~~~~~
+</errors>
```

Comment:
>errors [](http://example.com/codeflow?start=2&length=6)

Please also test with `Shared` only, not `Overridable`.

Reply:
> Please also test with Shared only, not Overridable.

Isn't this the previous test?